### DISCUSSION TITLE
ONNX.file

### DISCUSSION THREAD

**Discussion by Ragnar56** [onnx/models#465] [General]
Hi there

I have onnx. file which solves captchas. I want to link it to my web automation code but I do not have knowledge regarding onnx file and how I can link to my new app

Can you please help me with it?

Thanks

This message relates to: ONNX

**Comment by wenbingl** [comment]
Build a web application with ONNX Runtime

https://onnxruntime.ai/docs/reference/build-web-app.html

**Comment by tiagoeck** [comment]
Hi, 

How are you? Thank you for the articles and descriptions. Is there a possibility to infer about the model of the onnx from the file itself? 

Kind regards,

Tiago
